clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/unsecured/OAuthBearerUnsecuredLoginCallbackHandler.java

```diff
@@ -52,6 +56,10 @@
  * name and value except '{@code iat}' and '{@code exp}', both of which are
  * calculated automatically.
  * <p>
+ * <p>
```

Comment:
Probably good to state above "A {@code CallbackHandler} that recognizes {@link OAuthBearerTokenCallback} to return an unsecured OAuth 2 bearer token and {@link SaslExtensionsCallback} to return  SASL extensions."